python/marvin/tools/map.py

```diff
@@ -252,7 +287,7 @@ def restore(cls, path, delete=False):
 
         return marvin.core.marvin_pickle.restore(path, delete=delete)
 
-    def plot(self, array='value', xlim=None, ylim=None, zlim=None,
+    def _plot(self, array='value', xlim=None, ylim=None, zlim=None,
```

Comment:
Does the old plot (now renamed _plot) still do anything?  If not, we should probably remove it entirely.